### PR TITLE
Revert "Remove cross-language assembly checks from typing tests. Re-e…

### PR DESCRIPTION
**Pull request by rchande**
…nable them for solution load tests."

This reverts commit ae6b19a491aeadf529ac35b08773bf6c13073d3b.

I played around with the 4-core machine and it appears that the new base drop no longer loads all our modules to reload settings after the perf tests run the ResetRoslynOptions test.

@pharring @heejaechang

### CODE REVIEW

File: src/EditorFeatures/CSharpTest/PerfTests/CSharpPerfSolutionLoad.xml [hunk 1]
```diff
@@ -23,8 +23,8 @@
       <WaitForIdleCPU />
       <MeasureTimeStop />
 
-      <VerifyRoslynModulesLoadedStatus ExpectedStatus="CSharp" />
-      <VerifyRoslynModulesLoadedStatus ExpectedStatus="NoBasic" />
+      <!--<VerifyRoslynModulesLoadedStatus ExpectedStatus="CSharp" />
```
Comment (pharring) [inline src/EditorFeatures/CSharpTest/PerfTests/CSharpPerfSolutionLoad.xml]: I think you should leave these enabled. (i.e. don't revert this file)

File: src/EditorFeatures/VisualBasicTest/PerfTests/BasicPerfSolutionLoad.xml [hunk 2]
```diff
@@ -23,8 +23,8 @@
       <WaitForIdleCPU />
       <MeasureTimeStop />
 
-      <VerifyRoslynModulesLoadedStatus ExpectedStatus="NoCSharp" />
-      <VerifyRoslynModulesLoadedStatus ExpectedStatus="Basic" />
+      <!--<VerifyRoslynModulesLoadedStatus ExpectedStatus="NoCSharp" />
```
Comment (pharring) [inline src/EditorFeatures/VisualBasicTest/PerfTests/BasicPerfSolutionLoad.xml]: Same here. Leave it the way it is.

**Comment by pharring** [comment]
:+1: for master.

File: src/EditorFeatures/VisualBasicTest/PerfTests/BasicPerfSolutionLoad.xml (hunk 2 above)
Comment (rchande) [inline src/EditorFeatures/VisualBasicTest/PerfTests/BasicPerfSolutionLoad.xml]: Good catch. Thanks.

**Comment by heejaechang** [comment]
:+1:

**Pull request merged**